KelynPNjeri/Store-Manager
# #161041527 Create View Minimum Details Required.

#### What does this PR do?
This PR allows a Store Owner to view the minimum amount of a product that must be in the inventory at any given time.

#### Description of Task to be completed?
The task to be completed was to create a means by which a Store Owner could see the minimum amount of a product that must be in the inventory at any given.

#### How should this be manually tested?
You can clone into my repository using the git clone `git clone https://github.com/KelynPNjeri/Store-Manager.git` to your desired location.
Go to the location to which you cloned the repository.
Change directory into the Store Manager directory using `cd Store-Manager`
Navigate to the ft-minimum-item-required-161041527 branch using the `git checkout ft-minimum-item-required-161041527` command.
Launch the `product.html` file in the html folder using your favorite web browser.
Click on any product's image to see the details of a product among which you'll find the minimum item required in stock.

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
[#161041527](https://www.pivotaltracker.com/story/show/161041527)

#### Screenshots (if appropriate)
![min](https://user-images.githubusercontent.com/34427048/46835468-f00e2180-cdb6-11e8-8b91-e928fe2f847b.png)
